client/network/src/service/out_events.rs

```diff
@@ -52,10 +52,10 @@ use std::{
 ///
 /// The name is used in Prometheus reports, the queue size threshold is used
 /// to warn if there are too many unprocessed events in the channel.
-pub fn channel(name: &'static str, queue_size_warning: i64) -> (Sender, Receiver) {
+pub fn channel(name: &'static str, queue_size_warning: usize) -> (Sender, Receiver) {
```

Comment:
Should this go one step further and use `NonZeroUsize` as `0` doesn't really make much sense as a value here?

Reply:
This would require writing `NonZeroUsize::new(100_000).unwrap()` instead of `100_000` every time when passing the queue size threshold, so I'm not sure that it makes sense.